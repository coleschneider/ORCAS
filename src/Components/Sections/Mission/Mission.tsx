import * as React from 'react';
import { createSection } from 'utils/sectionUtils';
import Reveal from 'react-reveal';

import './mission.scss';

const MissionWrap = createSection({
  description: `Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
  assumenda repudiandae non cumque qui. Amet repellat omnis ea.`,
  displayTitle: 'Our Mission',
  sectionName: 'mission',
});

export default () => (
  <MissionWrap>
    <Reveal bottom={true}>
      <div className="row wide mission-desc">
        <div className="col-full slick-slider mission-desc__slider">
          <div className="mission-desc__slide">
            <h3 className="item-title">Community.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>
          <div className="mission-desc__slide">
            <h3 className="item-title">Family.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>
          <div className="mission-desc__slide">
            <h3 className="item-title">Secure.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>

          <div className="mission-desc__slide">
            <h3 className="item-title">Family.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>
        </div>
      </div>
    </Reveal>
  </MissionWrap>
);
