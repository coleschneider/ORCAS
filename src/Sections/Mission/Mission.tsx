import * as React from 'react';
import Reveal from 'react-reveal/Reveal';

export default () => (
  <section id="mission" className="s-about target-section">
    <Reveal top={true} cascade={true}>
      <div className="row section-header has-bottom-sep">
        <div className="col-full">
          <h1 className="display-1">Our Mission</h1>
          <p className="lead">
            Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
            assumenda repudiandae non cumque qui. Amet repellat omnis ea.
          </p>
        </div>
      </div>
      <div className="row wide about-desc">
        <div className="col-full slick-slider about-desc__slider">
          <div className="about-desc__slide">
            <h3 className="item-title">Community.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>
          <div className="about-desc__slide">
            <h3 className="item-title">Family.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>
          <div className="about-desc__slide">
            <h3 className="item-title">Secure.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>

          <div className="about-desc__slide">
            <h3 className="item-title">Family.</h3>
            <p>
              Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia
              assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
            </p>
          </div>
        </div>
      </div>
    </Reveal>
  </section>
);
