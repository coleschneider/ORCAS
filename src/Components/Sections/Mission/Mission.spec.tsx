import * as React from 'react';
import Mission from './Mission';

describe('Mission', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<Mission />);
    expect(wrapper).toMatchSnapshot();
  });
});
