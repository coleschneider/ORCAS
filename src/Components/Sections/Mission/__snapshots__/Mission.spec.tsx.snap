// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission matches the snapshot 1`] = `
<Component>
  <Reveal
    bottom={true}
    count={1}
    countOut={1}
    delay={0}
    delayOut={0}
    duration={1000}
    durationOut={1000}
    inEffect={
      Object {
        "count": 1,
        "delay": 0,
        "duration": 1000,
        "forever": undefined,
        "make": [Function],
        "reverse": undefined,
        "style": Object {
          "animationFillMode": "both",
        },
      }
    }
    outEffect={
      Object {
        "count": 1,
        "delay": 0,
        "duration": 1000,
        "forever": undefined,
        "make": [Function],
        "reverse": undefined,
        "style": Object {
          "animationFillMode": "both",
        },
      }
    }
  >
    <div
      className="row wide mission-desc"
    >
      <div
        className="col-full slick-slider mission-desc__slider"
      >
        <div
          className="mission-desc__slide"
        >
          <h3
            className="item-title"
          >
            Community.
          </h3>
          <p>
            Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
          </p>
        </div>
        <div
          className="mission-desc__slide"
        >
          <h3
            className="item-title"
          >
            Family.
          </h3>
          <p>
            Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
          </p>
        </div>
        <div
          className="mission-desc__slide"
        >
          <h3
            className="item-title"
          >
            Secure.
          </h3>
          <p>
            Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
          </p>
        </div>
        <div
          className="mission-desc__slide"
        >
          <h3
            className="item-title"
          >
            Family.
          </h3>
          <p>
            Et nihil atque ex. Reiciendis et rerum ut voluptate. Omnis molestiae nemo est. Ut quis enim rerum quia assumenda repudiandae non cumque qui. Amet repellat omnis ea aut cumque eos.
          </p>
        </div>
      </div>
    </div>
  </Reveal>
</Component>
`;
